seed/utils/api_schema.py

```diff
@@ -1,90 +1,80 @@
+    # Used to easily build out example values displayed on Swagger page.
+    body_parameter_formats = {
+        'interger_list': openapi.Schema(
+            type=openapi.TYPE_ARRAY,
+            items=openapi.Schema(type=openapi.TYPE_INTEGER)
+        )
+    }
+
+    def base_field(self, name, location_attr, description, required, type):
         """
-        Created to avoid needing to directly access coreapi within ViewSets.
+        Created to avoid needing to directly access openapi within ViewSets.
         Ideally, the cases below will be used instead of this one.
         """
-        return coreapi.Field(
+        return openapi.Parameter(
             name,
-            location=location,
+            getattr(openapi, location_attr),
+            description=description,
             required=required,
-            type=type,
-            description=description
+            type=type
         )
 
-    def org_id_field(self, location='query', required=True, type='integer', description='Organization ID'):
-        return coreapi.Field(
+    def org_id_field(self):
+        return openapi.Parameter(
             'organization_id',
-            location=location,
-            required=required,
-            type=type,
-            description=description
+            openapi.IN_QUERY,
+            description='Organization ID',
+            required=True,
+            type=openapi.TYPE_INTEGER
         )
 
-    def path_id_field(self, description):
-        return coreapi.Field(
-            "id",  # matches reference name in uri path
-            location='path',
+    def query_integer_field(self, name, description, required):
+        return openapi.Parameter(
+            name,
+            openapi.IN_QUERY,
+            description="Task ID created when DataQuality task is created.",
             required=True,
-            type='integer',
-            description=description
+            type=openapi.TYPE_INTEGER
         )
 
-    def body_field(self, required, description):
-        return coreapi.Field(
-            'body',
-            location='body',
-            required=required,
-            type='object',
-            description=description
+    def path_id_field(self, description):
+        return openapi.Parameter(
+            'id',
+            openapi.IN_PATH,
+            description=description,
+            required=True,
+            type=openapi.TYPE_INTEGER
         )
 
-    def form_field(self, name, required, description):
-        return coreapi.Field(
+    def body_field(self, required, description, name='body', params_to_formats={}):
+        return openapi.Parameter(
             name,
-            location='form',
+            openapi.IN_BODY,
+            description=description,
             required=required,
-            type='object',
-            description=description
+            schema=self._build_body_schema(params_to_formats)
```

Comment:
The starting point for building out the  "default value" on the Swagger pager as mentioned in [the other comment](https://github.com/SEED-platform/seed/pull/2188/files#r414104355). 

This will likely evolve/change as we try to apply the same type of "default value" to other endpoints with body params. In someway, we might also just give ourselves direct access to the openapi library to be able to build these out directly in the -Schema classes.

<img width="837" alt="Screen Shot 2020-04-23 at 2 26 36 PM" src="https://user-images.githubusercontent.com/30608004/80146544-3da04880-856f-11ea-8497-afbed87abe66.png">